include/libopencm3/stm32/g4/timer.h

```diff
@@ -29,3 +29,51 @@
 
 #pragma once
 #include <libopencm3/stm32/common/timer_common_f24.h>
+
+/* --- TIMx_CR2 values ----------------------------------------------------- */
+
+/****************************************************************************/
+/** @defgroup tim_x_cr2_ois TIMx_CR2_OIS: Force Output Idle State Control Values
+@{*/
+/* OIS4N:*//** Output idle state 4 (OC4N output) */
+#define TIM_CR2_OIS4N			(1 << 15)
+
+/* OIS5:*//** Output idle state 5 (OC5 output) */
+#define TIM_CR2_OIS4N			(1 << 16)
+
+/* OIS6:*//** Output idle state 6 (OC6 output) */
+#define TIM_CR2_OIS4N			(1 << 18)
+#define TIM_CR2_OIS_MASK		(0xfff << 8)
+/**@}*/
+
+/* MMS[3:0]: Master mode selection */
+/****************************************************************************/
+/** @defgroup tim_mastermode TIMx_CR2 MMS[25:4]: Master Mode Selection
+@{*/
+#define TIM_CR2_MMS_RESET		(0x0 << 4)
+#define TIM_CR2_MMS_ENABLE		(0x1 << 4)
+#define TIM_CR2_MMS_UPDATE		(0x2 << 4)
+#define TIM_CR2_MMS_COMPARE_PULSE	(0x3 << 4)
+#define TIM_CR2_MMS_COMPARE_OC1REF	(0x4 << 4)
+#define TIM_CR2_MMS_COMPARE_OC2REF	(0x5 << 4)
+#define TIM_CR2_MMS_COMPARE_OC3REF	(0x6 << 4)
+#define TIM_CR2_MMS_COMPARE_OC4REF	(0x7 << 4)
+#define TIM_CR2_MMS_ENCODER_OUTPUT	(0x200000 << 4)
+#define TIM_CR2_MMS_MASK		(0x200007 << 4)
```

Comment:
Standard style for the library is to have defines unshifted, and use a _SHIFT define where required.